nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -197,9 +197,141 @@ def test_plot_surf_roi_error():
     mesh = generate_surf()
     rng = np.random.RandomState(0)
     roi_idx = rng.randint(0, mesh[0].shape[0], size=5)
-
-    # Wrong input
     with pytest.raises(
             ValueError,
             match='roi_map does not have the same number of vertices'):
         plot_surf_roi(mesh, roi_map=roi_idx)
+
+
+def _generate_img():
+    mni_affine = MNI152TEMPLATE.get_affine()
+    data_positive = np.zeros((7, 7, 3))
+    rng = np.random.RandomState(42)
+    data_rng = rng.rand(7, 7, 3)
+    data_positive[1:-1, 2:-1, 1:] = data_rng[1:-1, 2:-1, 1:]
+    nii = nibabel.Nifti1Image(data_positive, mni_affine)
+    return nii
+
+
+def test_plot_img_on_surf_hemispheres_and_orientations():
```

Comment:
what are these tests testing?

Reply:
I think they could try to check some elements of the figure, at least the number of axes. at the moment despite the comment they don't check anything